src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft Azure PowerBI Dedicated Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for PowerBI Dedicated Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
Assembly file and assembly version should match with nuget version at least when you are not non-preview yet.
Once you become preview, only AssemblyFile version matches nuget version and AssemblyVersion revs up once you update major version.